Docs/source/running_cpp/parameters.rst

```diff
@@ -974,6 +974,14 @@ Numerics and algorithms
     This requires `warpx.use_kspace_filter=1` and is only supported
     with the RZ spectral solver.
 
+* ``warpx.use_damp_fields_in_z_guard`` (`0` or `1`; default: `0`)
```

Comment:
@dpgrote Thanks a lot for introducing with the last commit the additional logic for the default damping! I think it would be then necessary to remove the default value mentioned here and explain that it defaults to false always, except for RZ simulations that are not periodic in r.